ui/apps/platform/src/Containers/Risk/SecurityContext.js

```diff
@@ -25,14 +25,12 @@ const SecurityContext = ({ deployment }) => {
             .filter((container) => container.securityContext)
             .map((container) => {
                 const data = getSecurityContext(container);
-                if (data === {}) {
+                if (data === null || !Object.values(data).some((value) => !!value)) {
```

Comment:
This is a great find from the new rule!

Not a request for change, but we could potentially use lodash for clarity: 
```ts
if (isEmpty(data)) {
```

Reply:
Although I did not test, the following looked as if it might create an object that has properties whose values are `undefined` versus an empty object. So original code might have had double problem.

```js
const getSecurityContext = (container) => {
    if (!container.securityContext) {
        return null;
    }
    const { privileged, add_capabilities, drop_capabilities } = container.securityContext;
    return { privileged, add_capabilities, drop_capabilities };
};
```